File: servicces/src/doan.ProjectManagement.Application.Contracts/ProjectInformations/Dto/GetProjectInformationForInputDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace doan.ProjectManagement.ProjectInformations.Dto
{
    public class GetProjectInformationForInputDto : PagedAndSortedResultRequestDto
    {
        public string Filter { get; set; }
        public Guid? ProjectId { get; set; }
    }
}

File: applications/admin-web/src/app/shared/services/project/project-information/models/get-project-information-for-input-dto.ts
import { PagedAndSortedResultRequestDto } from '@abp/ng.core';

export class GetProjectInformationForinputDto extends PagedAndSortedResultRequestDto {
  filter: string;
  projectId: string;

  constructor(initial: Partial<GetProjectInformationForinputDto> = {}) {
    super(initial);
  }
}
